certified-connectors/PartnerLinq/apiDefinition.swagger.json

```diff
@@ -0,0 +1,190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PartnerLinq",
+    "description": "PartnerLinq is an integration tool used to connect two or more systems together. Use this connector to connect with PartnerLinq.",
+    "version": "1.0",
+	"contact":{
+		"name":"PartnerLinq Support",
+		"url":"https://www.partnerlinq.com/",
+		"email":"PL.Supportteam@visionet.com"
+	}
+  },
+  "host": "pod-dev1-cus-func-httpdatareceive.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/FUNC_HTTP_DATA_SEND": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "PartnerLinq Get",
+        "description": "PartnerLinq Get",
```

Comment:
@FarhanJamilVSI I would suggest you have different descriptions than the summaries. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).